packages/sdk-react-native/README.md

```diff
@@ -0,0 +1,145 @@
+# MetaMask SDK React Native
+
+The MetaMask SDK React Native allows developers to integrate MetaMask seamlessly into React Native applications.
+
+## Getting Started
+
+### Installation
+
+Install the SDK:
+
+```sh
+yarn add @metamask/sdk-react-native
+```
+
+### iOS Setup
+
+1. Add the following imports to your `AppDelegate.m`:
+
+```objective-c
+#import <React/RCTBundleURLProvider.h>
+#import <React/RCTBridge.h>
+#import <React/RCTLinkingManager.h>
+```
+
+2. Implement the following method in `AppDelegate.m` to handle deep links:
+
+```objective-c
+- (BOOL)application:(UIApplication *)app openURL:(NSURL *)url options:(NSDictionary<UIApplicationOpenURLOptionsKey,id> *)options
+{
+  return [RCTLinkingManager application:app openURL:url options:options];
+}
+```
+
+3. Add the URL scheme to your Xcode project:
+
+   - Open your project in Xcode.
+   - Go to the `Info` tab.
+   - Under `URL Types`, click the `+` button to add a new URL type.
+   - In the `URL Schemes` field, add your custom scheme (e.g., `yourappscheme`).
+
+4. Update your `Podfile` to include the following lines:
+```ruby
+# This is required in order to install the "metamask-ios-sdk" pod
+use_frameworks!
+pod 'metamask-ios-sdk', :git => 'https://github.com/MetaMask/metamask-ios-sdk.git'
+```
+
+4. Install CocoaPods dependencies:
+
+```sh
+cd ios && pod install
+```
+
+### Wrap Your Application in the Provider
+
+```js
+import React from 'react';
+import { AppRegistry } from 'react-native';
+import App from './App';
+import { MetaMaskProvider } from '@metamask/sdk-react-native';
+
+const sdkOptions = {
+  dappMetadata: {
+    name: 'Demo React Native App',
+    url: 'https://yourdapp.com',
+    iconUrl: 'https://yourdapp.com/icon.png',
+    scheme: 'yourappscheme',
+  },
+  infuraAPIKey: 'YOUR_INFURA_API_KEY', // Optional
```

Comment:
I would change the comment to say "Optional but highly recommended for a better user experience"